zaptest/timeout.go

```diff
@@ -21,31 +21,17 @@
 package zaptest
 
 import (
-	"log"
-	"os"
-	"strconv"
 	"time"
-)
 
-var _timeoutScale = 1.0
+	"go.uber.org/zap/internal/ztest"
+)
 
 // Timeout scales the provided duration by $TEST_TIMEOUT_SCALE.
 func Timeout(base time.Duration) time.Duration {
-	return time.Duration(float64(base) * _timeoutScale)
+	return ztest.Timeout(base)
 }
 
 // Sleep scales the sleep duration by $TEST_TIMEOUT_SCALE.
```

Comment:
same as above, should this be deprecated?

Reply:
Yep.